docs/root/faq/load_balancing/concurrency_lb.rst

```diff
@@ -7,5 +7,5 @@ load balancing policies such as :ref:`round robin <arch_overview_load_balancing_
 it may thus appear that load balancing is not working properly when using multiple workers. The
 :option:`--concurrency` option can be used to adjust the number of workers if desired.
 
-The siloed execution model is also the reason why multiple HTTP/2 connections may be established to
+The siloed execution model is also the reason why multiple HTTP/2 or HTTP/3 connections may be established to
```

Comment:
This is not a question about this PR, but it does make me wonder.... Presumably EnvoyMobile will be a single silo? (Maybe that was in the presentation. I should go re-read those slides)